BUILD.bazel

```diff
@@ -525,16 +551,17 @@ genrule(
         cp $(location python/ray/_raylet.so) python/ray &&
         mkdir -p python/ray/core/src/ray/thirdparty/redis/src/ &&
         cp $(location //:redis-server) python/ray/core/src/ray/thirdparty/redis/src/ &&
+        cp $(location //:redis-cli) python/ray/core/src/ray/thirdparty/redis/src/ &&
         mkdir -p python/ray/core/src/ray/gcs/redis_module/ &&
-        for f in $(locations //:ray_redis_module); do cp $$f python/ray/core/src/ray/gcs/redis_module/; done &&
+        cp $(locations //:ray_redis_module.so) python/ray/core/src/ray/gcs/redis_module/libray_redis_module.so &&
         mkdir -p python/ray/core/src/ray/raylet/ &&
         cp $(location //:raylet_monitor) python/ray/core/src/ray/raylet/ &&
         mkdir -p python/ray/core/src/plasma &&
         cp $(location @plasma//:plasma_store_server) python/ray/core/src/plasma/ &&
         cp $(location //:raylet) python/ray/core/src/ray/raylet/ &&
-        mv python $(location ray_pkg) &&
-        mkdir -p $(location ray_pkg)/java_lib &&
-        cp $(location @plasma//:plasma_client_java) $(location ray_pkg)/java_lib &&
```

Comment:
@guoyuhong We probably need a flag of some sort that only compiles the java extensions if the jdk is installed. Let's do this when we switch the jdk build over to bazel!

Reply:
@pcmoritz That is fine for me~